2015/02/25 15:34:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "IntITA".' in D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('IntITA')
#1 D:\xampp\htdocs\IntITA\protected\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 D:\xampp\htdocs\IntITA\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/IntITA/
---
2015/02/25 15:47:06 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'carousel.carousel_id' in 'field list'. The SQL statement executed was: SELECT `carousel`.`carousel_id`, `carousel`.`order`, `carousel`.`picture_url`, `carousel`.`width`, `carousel`.`height`, `carousel`.`description`
FROM `carousel`
ORDER BY `carousel`.`order` LIMIT 80.
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\models\AADb.php (97)
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php (612)
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php (564)
2015/02/25 15:47:06 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'carousel.carousel_id' in 'field list'. The SQL statement executed was: SELECT `carousel`.`carousel_id`, `carousel`.`order`, `carousel`.`picture_url`, `carousel`.`width`, `carousel`.`height`, `carousel`.`description`
FROM `carousel`
ORDER BY `carousel`.`order` LIMIT 80' in D:\xampp\htdocs\IntITA\protected\framework\db\CDbCommand.php:543
Stack trace:
#0 D:\xampp\htdocs\IntITA\protected\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\models\AADb.php(97): CDbCommand->queryAll()
#2 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(612): AADb->getList(80, 0)
#3 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(564): AutoAdmin->listMode()
#4 D:\xampp\htdocs\IntITA\protected\modules\autoAdmin\controllers\MainpageController.php(44): AutoAdmin->process()
#5 D:\xampp\htdocs\IntITA\protected\framework\web\actions\CInlineAction.php(49): MainpageController->actionCarousel()
#6 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(282): CController->run('carousel')
#10 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('autoAdmin/mainp...')
#11 D:\xampp\htdocs\IntITA\protected\framework\base\CApplication.php(184): CWebApplication->processRequest()
#12 D:\xampp\htdocs\IntITA\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/IntITA/index.php?r=autoAdmin/mainpage/carousel/
HTTP_REFERER=http://localhost:63342/IntITA/index.php?r=autoAdmin
---
2015/02/25 15:48:29 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '3' for key 'PRIMARY'. The SQL statement executed was: UPDATE `carousel` SET `order`=:order WHERE (carousel.order = :_idorder).
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\models\AADb.php (343)
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php (816)
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php (495)
2015/02/25 15:48:29 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '3' for key 'PRIMARY'. The SQL statement executed was: UPDATE `carousel` SET `order`=:order WHERE (carousel.order = :_idorder)' in D:\xampp\htdocs\IntITA\protected\framework\db\CDbCommand.php:358
Stack trace:
#0 D:\xampp\htdocs\IntITA\protected\framework\db\CDbCommand.php(1313): CDbCommand->execute(Array)
#1 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\models\AADb.php(343): CDbCommand->update('carousel', Array, Array, Array)
#2 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(816): AADb->update(Array)
#3 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(495): AutoAdmin->update()
#4 D:\xampp\htdocs\IntITA\protected\modules\autoAdmin\controllers\MainpageController.php(44): AutoAdmin->process()
#5 D:\xampp\htdocs\IntITA\protected\framework\web\actions\CInlineAction.php(49): MainpageController->actionCarousel()
#6 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(282): CController->run('carousel')
#10 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('autoAdmin/mainp...')
#11 D:\xampp\htdocs\IntITA\protected\framework\base\CApplication.php(184): CWebApplication->processRequest()
#12 D:\xampp\htdocs\IntITA\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/IntITA/?r=autoAdmin/mainpage/carousel/&id[order]=1&action=update
HTTP_REFERER=http://localhost:63342/IntITA/?r=autoAdmin/mainpage/carousel/&action=edit&id[order]=1
---
2015/02/25 15:55:13 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'int_ita_db.block' doesn't exist. The SQL statement executed was: SELECT `block`.`block_id`, `block`.`header`, `block`.`picture_url`, `block`.`annotation`, `block`.`link`
FROM `block`
ORDER BY `block`.`header` LIMIT 80.
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\models\AADb.php (97)
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php (612)
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php (564)
2015/02/25 15:55:13 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'int_ita_db.block' doesn't exist. The SQL statement executed was: SELECT `block`.`block_id`, `block`.`header`, `block`.`picture_url`, `block`.`annotation`, `block`.`link`
FROM `block`
ORDER BY `block`.`header` LIMIT 80' in D:\xampp\htdocs\IntITA\protected\framework\db\CDbCommand.php:543
Stack trace:
#0 D:\xampp\htdocs\IntITA\protected\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\models\AADb.php(97): CDbCommand->queryAll()
#2 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(612): AADb->getList(80, 0)
#3 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(564): AutoAdmin->listMode()
#4 D:\xampp\htdocs\IntITA\protected\modules\autoAdmin\controllers\MainpageController.php(62): AutoAdmin->process()
#5 D:\xampp\htdocs\IntITA\protected\framework\web\actions\CInlineAction.php(49): MainpageController->actionBlocks()
#6 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(282): CController->run('blocks')
#10 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('autoAdmin/mainp...')
#11 D:\xampp\htdocs\IntITA\protected\framework\base\CApplication.php(184): CWebApplication->processRequest()
#12 D:\xampp\htdocs\IntITA\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/IntITA/index.php?r=autoAdmin/mainpage/blocks/
HTTP_REFERER=http://localhost:63342/IntITA/index.php?r=autoAdmin
---
2015/02/25 15:56:22 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'aboutus.header' in 'field list'. The SQL statement executed was: SELECT `aboutus`.`block_id`, `aboutus`.`header`, `aboutus`.`picture_url`, `aboutus`.`annotation`, `aboutus`.`link`
FROM `aboutus`
ORDER BY `aboutus`.`header` LIMIT 80.
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\models\AADb.php (97)
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php (612)
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php (564)
2015/02/25 15:56:22 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'aboutus.header' in 'field list'. The SQL statement executed was: SELECT `aboutus`.`block_id`, `aboutus`.`header`, `aboutus`.`picture_url`, `aboutus`.`annotation`, `aboutus`.`link`
FROM `aboutus`
ORDER BY `aboutus`.`header` LIMIT 80' in D:\xampp\htdocs\IntITA\protected\framework\db\CDbCommand.php:543
Stack trace:
#0 D:\xampp\htdocs\IntITA\protected\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\models\AADb.php(97): CDbCommand->queryAll()
#2 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(612): AADb->getList(80, 0)
#3 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(564): AutoAdmin->listMode()
#4 D:\xampp\htdocs\IntITA\protected\modules\autoAdmin\controllers\MainpageController.php(62): AutoAdmin->process()
#5 D:\xampp\htdocs\IntITA\protected\framework\web\actions\CInlineAction.php(49): MainpageController->actionBlocks()
#6 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(282): CController->run('blocks')
#10 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('autoAdmin/mainp...')
#11 D:\xampp\htdocs\IntITA\protected\framework\base\CApplication.php(184): CWebApplication->processRequest()
#12 D:\xampp\htdocs\IntITA\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/IntITA/index.php?r=autoAdmin/mainpage/blocks/
HTTP_REFERER=http://localhost:63342/IntITA/index.php?r=autoAdmin
---
2015/02/25 15:58:22 [error] [exception.AAException] exception 'AAException' with message 'Wrong field name "header" for sorting' in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\models\AAData.php:135
Stack trace:
#0 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(923): AAData->setSortOrder(Array)
#1 D:\xampp\htdocs\IntITA\protected\modules\autoAdmin\controllers\MainpageController.php(61): AutoAdmin->sortDefault(Array)
#2 D:\xampp\htdocs\IntITA\protected\framework\web\actions\CInlineAction.php(49): MainpageController->actionBlocks()
#3 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(282): CController->run('blocks')
#7 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('autoAdmin/mainp...')
#8 D:\xampp\htdocs\IntITA\protected\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 D:\xampp\htdocs\IntITA\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/IntITA/index.php?r=autoAdmin/mainpage/blocks/
HTTP_REFERER=http://localhost:63342/IntITA/index.php?r=autoAdmin
---
2015/02/25 16:00:02 [error] [php] Undefined index: autoadmin (D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\helpers\AAHelperFile.php:16)
Stack trace:
#0 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(891): AAFieldImage->loadFromForm()
#1 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(782): AutoAdmin->fillFieldsWithForm()
#2 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(495): AutoAdmin->update()
#3 D:\xampp\htdocs\IntITA\protected\modules\autoAdmin\controllers\MainpageController.php(62): AutoAdmin->process()
#4 D:\xampp\htdocs\IntITA\protected\framework\web\actions\CInlineAction.php(49): MainpageController->actionBlocks()
#5 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(286): MainpageController->runAction()
#7 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(265): MainpageController->runActionWithFilters()
#8 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(282): MainpageController->run()
#9 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 D:\xampp\htdocs\IntITA\protected\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 D:\xampp\htdocs\IntITA\index.php(13): CWebApplication->run()
REQUEST_URI=/IntITA/?r=autoAdmin/mainpage/blocks/&sortBy=1&id[block_id]=1&action=update
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\helpers\AAHelperFile.php (16)
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\helpers\AAHelperFile.php (57)
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\models\field\AAFieldFile.php (101)
2015/02/25 16:01:04 [error] [php] Undefined index: autoadmin (D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\helpers\AAHelperFile.php:16)
Stack trace:
#0 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(891): AAFieldImage->loadFromForm()
#1 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(782): AutoAdmin->fillFieldsWithForm()
#2 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(495): AutoAdmin->update()
#3 D:\xampp\htdocs\IntITA\protected\modules\autoAdmin\controllers\MainpageController.php(62): AutoAdmin->process()
#4 D:\xampp\htdocs\IntITA\protected\framework\web\actions\CInlineAction.php(49): MainpageController->actionBlocks()
#5 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(286): MainpageController->runAction()
#7 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(265): MainpageController->runActionWithFilters()
#8 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(282): MainpageController->run()
#9 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 D:\xampp\htdocs\IntITA\protected\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 D:\xampp\htdocs\IntITA\index.php(13): CWebApplication->run()
REQUEST_URI=/IntITA/?r=autoAdmin/mainpage/blocks/&sortBy=1&id[block_id]=1&action=update
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\helpers\AAHelperFile.php (16)
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\helpers\AAHelperFile.php (57)
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\models\field\AAFieldFile.php (101)
2015/02/25 16:01:44 [error] [php] Undefined index: autoadmin (D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\helpers\AAHelperFile.php:16)
Stack trace:
#0 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(891): AAFieldImage->loadFromForm()
#1 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(782): AutoAdmin->fillFieldsWithForm()
#2 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(495): AutoAdmin->update()
#3 D:\xampp\htdocs\IntITA\protected\modules\autoAdmin\controllers\MainpageController.php(62): AutoAdmin->process()
#4 D:\xampp\htdocs\IntITA\protected\framework\web\actions\CInlineAction.php(49): MainpageController->actionBlocks()
#5 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(286): MainpageController->runAction()
#7 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(265): MainpageController->runActionWithFilters()
#8 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(282): MainpageController->run()
#9 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 D:\xampp\htdocs\IntITA\protected\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 D:\xampp\htdocs\IntITA\index.php(13): CWebApplication->run()
REQUEST_URI=/IntITA/?r=autoAdmin/mainpage/blocks/&sortBy=1&id[block_id]=1&action=update
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\helpers\AAHelperFile.php (16)
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\helpers\AAHelperFile.php (57)
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\models\field\AAFieldFile.php (101)
2015/02/25 16:03:20 [error] [php] Undefined index: autoadmin (D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\helpers\AAHelperFile.php:16)
Stack trace:
#0 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(891): AAFieldImage->loadFromForm()
#1 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(782): AutoAdmin->fillFieldsWithForm()
#2 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(495): AutoAdmin->update()
#3 D:\xampp\htdocs\IntITA\protected\modules\autoAdmin\controllers\MainpageController.php(62): AutoAdmin->process()
#4 D:\xampp\htdocs\IntITA\protected\framework\web\actions\CInlineAction.php(49): MainpageController->actionBlocks()
#5 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(286): MainpageController->runAction()
#7 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(265): MainpageController->runActionWithFilters()
#8 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(282): MainpageController->run()
#9 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 D:\xampp\htdocs\IntITA\protected\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 D:\xampp\htdocs\IntITA\index.php(13): CWebApplication->run()
REQUEST_URI=/IntITA/?r=autoAdmin/mainpage/blocks/&sortBy=1&id[block_id]=1&action=update
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\helpers\AAHelperFile.php (16)
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\helpers\AAHelperFile.php (57)
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\models\field\AAFieldFile.php (101)
2015/02/25 16:05:08 [error] [php] Undefined index: autoadmin (D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\helpers\AAHelperFile.php:16)
Stack trace:
#0 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(891): AAFieldImage->loadFromForm()
#1 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(782): AutoAdmin->fillFieldsWithForm()
#2 D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\AutoAdmin.php(495): AutoAdmin->update()
#3 D:\xampp\htdocs\IntITA\protected\modules\autoAdmin\controllers\MainpageController.php(62): AutoAdmin->process()
#4 D:\xampp\htdocs\IntITA\protected\framework\web\actions\CInlineAction.php(49): MainpageController->actionBlocks()
#5 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(286): MainpageController->runAction()
#7 D:\xampp\htdocs\IntITA\protected\framework\web\CController.php(265): MainpageController->runActionWithFilters()
#8 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(282): MainpageController->run()
#9 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 D:\xampp\htdocs\IntITA\protected\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 D:\xampp\htdocs\IntITA\index.php(13): CWebApplication->run()
REQUEST_URI=/IntITA/?r=autoAdmin/mainpage/blocks/&id[block_id]=1&action=update
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\helpers\AAHelperFile.php (16)
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\helpers\AAHelperFile.php (57)
in D:\xampp\htdocs\IntITA\protected\extensions\autoAdmin\models\field\AAFieldFile.php (101)
2015/02/25 16:09:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "aaauth/login".' in D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('aaauth/login')
#1 D:\xampp\htdocs\IntITA\protected\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 D:\xampp\htdocs\IntITA\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/IntITA/index.php?r=aaauth/login
---
2015/02/25 16:17:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "aaauth/login".' in D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('aaauth/login')
#1 D:\xampp\htdocs\IntITA\protected\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 D:\xampp\htdocs\IntITA\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/IntITA/index.php?r=aaauth/login
---
2015/02/25 16:19:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "aaauth/login".' in D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('aaauth/login')
#1 D:\xampp\htdocs\IntITA\protected\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 D:\xampp\htdocs\IntITA\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/IntITA/index.php?r=aaauth/login
---
2015/02/25 16:20:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "aaauth/login".' in D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('aaauth/login')
#1 D:\xampp\htdocs\IntITA\protected\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 D:\xampp\htdocs\IntITA\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/IntITA/index.php?r=aaauth/login
---
2015/02/25 16:40:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "aaauth/login".' in D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('aaauth/login')
#1 D:\xampp\htdocs\IntITA\protected\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 D:\xampp\htdocs\IntITA\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/IntITA/index.php?r=aaauth/login
---
2015/02/25 16:46:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "aaauth/login".' in D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\IntITA\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('aaauth/login')
#1 D:\xampp\htdocs\IntITA\protected\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 D:\xampp\htdocs\IntITA\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/IntITA//index.php?r=aaauth/login
---
